hal/src/uhura/include.mk

```diff
@@ -0,0 +1,19 @@
+
+# Define the prefix to this directory.
+# Note: The name must be unique within this build and should be
+#       based on the root of the project
+
+INCLUDE_DIRS += $(TARGET_HAL_PATH)/src/$(PLATFORM_NAME)
+# # FIXME: There is no NCP on TRACKERM??? check this please
```

Comment:
WiFi is onboard, but it's still driven by ncp client etc. See `network/lwip/realtek`.